browser/ephemeral_storage/ephemeral_storage_tab_helper.cc

```diff
@@ -9,6 +9,7 @@
 #include <memory>
 #include <set>
 
+#include "base/containers/contains.h"
```

Comment:
what is it for

Reply:
btw find_if could be simplified in this file

Reply:
Fixed the header inclusion.

Regarding `find_if`
```
  auto it = base::ranges::find_if(keep_alive_tld_ephemeral_lifetime_list_,
                                  [&key](const auto& tld_ephermal_liftime) {
                                    return tld_ephermal_liftime->key() == key;
                                  });
```
As you can see in this snippet, there's an indirection in `tld_ephermal_liftime->key()`, so we can't just access `key` with a projection.